framework/application/routes/index.ts

```diff
@@ -7,17 +7,14 @@ module.exports = () => {
   const router = Router();
 
   router.use(PaginateHandler);
+  router.use(bodyParser.json())
 
-  const apiRouter = Router();
-  apiRouter.use(bodyParser.json());
+  router.use('/healthcheck', (req, res, next) => {
 
-  apiRouter.use('/check', (req, res, next) => {
     return res
       .status(200)
-      .json({ message: `Hello, I'm attiv and I'm here to help you develop software faster, robust and secure.` });
+      .json({ message: `Hello, I'm attiv and i'm here to help you develop software faster, robust and secure. The process is running for ${Math.floor(process.uptime()) / 60} minutes` });
```

Comment:
There's another "I'm" that can be replaced by "i'm".
Another suggestion is keep uptime in seconds.

Reply:
It's not better keep it in minutes? Why do you prefer in seconds?

I think the number in seconds will be too big if all the things are running ok, making dificult to human readability

Reply:
I think is more common. With minutes we have the same problem when it's running for a long time, I think. Maybe add the startup time helps 🤔